dynawo/sources/API/PAR/PARReference.cpp

```diff
@@ -27,7 +27,14 @@ using std::string;
 
 namespace parameters {
 
-Reference::Reference(const string& name) : type_(""), name_(name), origName_(""), componentId_("") {}
+Reference::Reference(const string& name, OriginData origData) :
```

Comment:
We must choose if the reference has origData IIDM or PAR. Because we have to choose at the creation of the object, we have to initialize origData_ in the constructor. I think we may need to delete the setOrigData method as well.

Reply:
I mean: why do we need to specify it in the constructor when before it was not required?

Reply:
Before, IIDM was the only choice.

Reply:
Ok, so in this case let's not put a default value to the origData parameter and make sure that the setter is not used